qubespdfconverter/client.py

```diff
@@ -397,6 +397,11 @@ async def _save_reps(self, pages):
                     Image.open,
                     Path(self.pdf.parent, f"{page}.png"))
                 )
+                for image in images:
```

Comment:
This will execute `load()` for pages repeatedly, as it's inside the `for page in pages` loop. It should either be moved outside of this loop (and wrapped in separate try/except), or the earlier call instead of `images.append(await ...` should save image into a variable, call `load()` on it and only then append to `images`.